cmake/std/atdm/ride/environment.sh

```diff
@@ -57,38 +78,47 @@ else
   export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=32
 fi
 
-if [ "$ATDM_CONFIG_COMPILER" == "GNU" ]; then
-    module load devpack/20180308/openmpi/2.1.2/gcc/7.2.0/cuda/9.0.176
-    module swap openblas/0.2.20/gcc/7.2.0 netlib/3.8.0/gcc/7.2.0
-    export OMPI_CXX=`which g++`
-    export OMPI_CC=`which gcc`
-    export OMPI_FC=`which gfortran`
-    export ATDM_CONFIG_LAPACK_LIB="-L${LAPACK_ROOT}/lib;-llapack;-lgfortran;-lgomp"
-    export ATDM_CONFIG_BLAS_LIB="-L${BLAS_ROOT}/lib;-lblas;-lgfortran;-lgomp;-lm"
-elif [ "$ATDM_CONFIG_COMPILER" == "CUDA" ]; then
-    export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=16
-    module load devpack/20180308/openmpi/2.1.2/gcc/7.2.0/cuda/9.0.176
-    module unload gcc/7.2.0
+if [ "$ATDM_CONFIG_COMPILER" == "GNU" ] ; then
+
+  # Load the modules and set up env
+  module load devpack/20180308/openmpi/2.1.2/gcc/7.2.0/cuda/9.0.176
```

Comment:
Should this be cuda 9.2?

Reply:
@rppawlo, that is what is is currently.  But given this is a GNU build and does not use CUDA, I suspect that it should not matter.  This is still a GCC 7.2.0 build.  But we should likely change it I guess since the CUDA 9.0 env does not really work with CUDA.